.github/workflows/disable-flaky-tests.yml

```diff
@@ -0,0 +1,18 @@
+name: Disable Flaky Tests
+
+on:
+  schedule:
+    # At minute 39 past every 3rd hour Monday through Friday.
+    - cron: "39 */3 * * 1-5"
+  # Have the ability to trigger this job manually through the API
+  workflow_dispatch:
+
+jobs:
+  cron:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Fetch and file issues for flaky tests
+        run: |
+          curl --request POST \
+          --url 'https://www.torch-ci.com/api/flaky-tests/disable' \
+          --header 'Authorization: Bearer ${{ secrets.ACTION_KEY }}'
```

Comment:
Right now I'm fairly certain anyone can trigger this by just going to this url, which doesn't feel very safe. How can I add authorization on the server side?

Reply:
answered below